proxy-robots-module/src/main/java/orwell/proxy/Cli.java

```diff
@@ -0,0 +1,86 @@
+package orwell.proxy;
+
+import org.apache.commons.cli.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.ConfigCli;
+import orwell.proxy.config.EnumConfigFileType;
+
+import java.io.File;
+
+/**
+ * Created by miludmann on 5/5/15.
+ */
+public class Cli {
+    final static protected String DEFAULT_CONFIG_FILEPATH = "/configuration.xml";
```

Comment:
This is indeed a filepath used inside the jar itself, so no incidence on plateform.
Will rename it to CONFIG_FILEPATH_INSIDE_JAR.